src/desktop/package.json

```
@@ -117,15 +117,27 @@
     "nsis": {
       "deleteAppDataOnUninstall": true
     },
+    "snap": {
+      "stagePackages": [
+        "default",
+        "libsecret"
```

```suggestion
        "libsecret-1-dev"
```
From https://docs.snapcraft.io/adding-parts:
> Package names are those used by the build host’s package manager, such as apt or dnf.

Attempting to build Linux gives the following error:
```
Failed to fetch stage packages: Error downloading packages for part 'app': The package 'libsecret' was not found..
```